features/step_definitions/admin_category_steps.rb

```diff
@@ -226,8 +218,8 @@ def find_transaction_type_by_name(transaction_type_name)
 end
 
 When /^I toggle transaction type "(.*?)"$/ do |transaction_type_name|
-  transaction_type = find_transaction_type_by_name(transaction_type_name)
-  find(:css, "#transaction_type_checkbox_#{transaction_type.id}").click
+  shape = find_shape(name: transaction_type_name)
```

Comment:
Remember to fix finding by name. Now it breaks with two clashing names. 

So, don't allow same shape names for same locale inside a community.